cmake/kokkos_functions.cmake

```diff
@@ -792,6 +792,14 @@ FUNCTION(COMPILER_SPECIFIC_FLAGS)
   COMPILER_SPECIFIC_OPTIONS_HELPER(${ARGN} COMPILE_OPTIONS LINK_OPTIONS)
 ENDFUNCTION(COMPILER_SPECIFIC_FLAGS)
 
+FUNCTION(COMPILER_SPECIFIC_OPTIONS)
+  COMPILER_SPECIFIC_OPTIONS_HELPER(${ARGN} COMPILE_OPTIONS)
+ENDFUNCTION(COMPILER_SPECIFIC_OPTIONS)
+
+FUNCTION(COMPILER_SPECIFIC_LINK_OPTIONS)
+  COMPILER_SPECIFIC_OPTIONS_HELPER(${ARGN} LINK_OPTIONS)
+ENDFUNCTION(COMPILER_SPECIFIC_LINK_OPTIONS)
```

Comment:
Is this used at all?

Reply:
This function no, it's made to remain consistent